src/main/features/claim/routes/pay.ts

```diff
@@ -93,10 +95,10 @@ async function successHandler (res, next) {
       logger.error(`missing consent not given role for user, User Id : ${user.id}`)
     }
 
-    let features: string
-    if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_admissions')) {
+    let features: string = ''
+    await launchDarklyClient.callFeatureFlag(user, 'cmc_admissions' , () => {
```

Comment:
this isn't quite the same as before, the toggle is configured to only sometimes say yes and only if the user has the role `cmc-new-features-consent-given`

Reply:
I have added it to LDUser as custom attributes

Reply:
yes!!!!! sorry i forgot about that @klawani 🙈